lib/logger.js

```diff
@@ -96,8 +96,8 @@ logger.log = function (...args) {
 logger.banner = function () {
 	var info = appc.pkginfo.package(module, 'version', 'about');
 	if (bannerEnabled) {
-		logger.log(info.about.name.cyan.bold + ', CLI version ' + info.version + (logger.activeSdk ? ', Titanium SDK version ' + logger.activeSdk : '') + '\n' + info.about.copyright);
-		logger.log('\n' + __('Please report bugs to %s', 'https://github.com/tidev/titanium_mobile/issues'.cyan) + '\n');
+		logger.log('Titanium CLI v' + info.version + (logger.activeSdk ? ', SDK v' + logger.activeSdk : '') + ', ' + 'https://titaniumsdk.com'.cyan);
```

Comment:
Gah, I can't quite represent colors in GitHub markdown... can you please try making the "Titanium CLI" `.cyan.bold`?

Reply:
Like this?

<img width="514" alt="Bildschirmfoto 2022-05-19 um 07 50 03" src="https://user-images.githubusercontent.com/1126641/169219495-d2d1622e-9337-4b3d-8918-ff5fbf63177b.png">

Reply:
Dang, looking hot. Dunno if you're feeling it, but it feels lighter.

Now I suppose we resolve these conversations and solicit feedback from @joshualambert and @jasonkneen.